netket/optimizer/qgt/qgt_jacobian.py

```diff
@@ -24,11 +24,24 @@
 from netket.stats import sum_inplace
 import netket.jax as nkjax
 
-from .base import AbstractSMatrix
+from ..linear_operator import LinearOperator
+
+
+def QGTJacobian(vstate, *, mode, rescale_shift=False) -> "QGTJacobianT":
```

Comment:
@attila-i-szabo `rescale_shift` is a scale factor right? so it can be False to say you don't want rescaling or a number, right?

Then why not just call it `rescaling_factor` or `rescale_factor` and have it be `None` or a float? Anything against that? I think it's more consistent with the rest of the interface to have None instead of False for unused arguments by default.

Reply:
No, it is a switch whether to use scale-invariant regularisation or no. I picked the name from netket 2.0.

Reply:
ah, nevermind then

Reply:
Also, would you mind changing the default to `True`? It really should be used if it's there...